PLAYBOOKS.md

```diff
@@ -10,44 +10,21 @@ A playbook is a JSON file that will be used by our [scraping engine](./app/engin
-        {
-          "route": "/index-en-GB.html", // Route for the front index page.
-          "navigations": [ //The scraping engine selects all the selectors and get thier src and progressively make his way through it.
-            {
-              "selector": "frame[name=\"eAISNavigationBase\"]",
-              "attribute": "src"
-            },
-            // A selector can be anything that has a src or href to guide the engine
-            {
-              "selector": "frame[name=\"eAISNavigation\"]",
-              "attribute": "src"
-            }
-          ]
-        }
-      ],
-      // Then we look for aerodromes and airport charts. The aerodromes are identified by thier
-      // icao codes. We select a selector on basis of icao code.
-      "chart": {
-        "baseUrl": "${baseUrl}/eAIP/",
-        "selector": "a[title=\"${icao}\"]",
-        "attribute": "href"
-      }
-    }
-  }
-}
-```
+| Keys          | Description                                            | Data type        | Required | Parent key | Example                                                                                                                          |
+| ------------- | ------------------------------------------------------ | ---------------- | -------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| iso           | ISO code of country                                    | String           | Yes      | Country    | `"iso":"IN"`                                                                                                                     |
+| name          | Name of the country                                    | String           | Yes      | Country    | `"name": "India"`                                                                                                                |
+| source        | Source of information                                  | String           | Yes      | source     | `"source": "Airports Authority of India"`                                                                                        |
+| engine        | Name of scraping engine(Axios currently)               | String           | Yes      | scraper    | `"engine": "axios"`                                                                                                              |
+| baseUrl       | URL for charts website(eAIP)                           | String           | Yes      | features   | `"baseUrl": "https://aim-india.aai.aero/eaip-v2-02-2021"`                                                                        |
+| paths         | Path for getting charts from eAIP                      | Array            | Yes      | features   | [See example](https://github.com/drph4nt0m/avbot-charts/blob/ec58c14ae01da70bcff68a4360027a2142a80366/playbooks/AU.json#L11-L15) |
```

Comment:
Path is not `Path for getting charts from eAIP` but it defines multiple routes it can go to